### ISSUE TITLE
训练时是否支持网络多输出（网络输出多个prediction）？

### ISSUE THREAD

**Issue by CallMeEricChiu** [alibaba/MNN#935]
我准备用bisenetv2（图在下面）做移动端的语义分割训练，但这个网络结构呢训练时loss由五个部分组成，也就是这个网络要输出五个prediction来与gt计算总loss。
但是我看Module.cpp中的forward是这么定义的：
`Express::VARP Module::forward(Express::VARP input) {
    return this->onForward({input})[0];}`
这只能是单输出吧，如果要多输出就得自己把这个forward改了对吧？
另外网络结构我是从pytorch模型转成mnn模型的，pytorch的forward最后输出是下面这样，我转换到mnn后，若能多输出，模型输出的顺序是怎么样的呢？
`

    def forward(self, x):
        size = x.size()[2:]
        feat_d = self.detail(x)
        feat2, feat3, feat4, feat5_4, feat_s = self.segment(x)
        feat_head = self.bga(feat_d, feat_s)
        logits = self.head(feat_head, size)
        logits_aux2 = self.aux2(feat2, size)
        logits_aux3 = self.aux3(feat3, size)
        logits_aux4 = self.aux4(feat4, size)
        logits_aux5_4 = self.aux5_4(feat5_4, size)
        return logits, logits_aux2, logits_aux3, logits_aux4, logits_aux5_4`
网络结构是这样子的：
![image](https://user-images.githubusercontent.com/51788361/86233776-89ebb280-bbc8-11ea-9f6e-0d55de74f9e0.png)

**Comment by jxt1234** [comment]
Module 有一个 onForward 接口是多输入多输出的

如果是 python 传入 list 就是多输出版本

**Comment by CallMeEricChiu** [comment]
@jxt1234 感谢感谢
